docs/README.fedora

```diff
@@ -0,0 +1,78 @@
+TOC
+1. Introduction
+2. Getting the source code
+3. Installing the required libraries and headers
+4. How to compile
+5. Uninstalling
+
+-----------------------------------------------------------------------------
+1. Introduction
+-----------------------------------------------------------------------------
+
+A graphics-adapter with OpenGL or OpenGLES acceleration is required.
+
+Note to new Linux users:
+All lines that are prefixed with the '$' character are commands,
+that need to be typed into a terminal window / console. The '$' equals the prompt.
+Note: The '$' character itself should NOT be typed as part of the command.
+
+Note to new Fedora users:
+For downloading packages use, $ sudo dnf <package_name>
```

Comment:
This is wrong, and you don't need to mention this here since you write the install command further down